### DISCUSSION TITLE
Navigating folders from Mountain Duck mount extremely slow

### DISCUSSION THREAD

**Discussion by jballment** [iterate-ch/cyberduck#16322] [Q&A]
I have been using Mountain Duck for years and about 6 months ago I notice that it's become very very slow to navigate the folder within my nextcloud mounted director in Windows 10 using Mountain Duck 4.16.2 (22310).  We have a very large nextcloud instance, 1000's of folders and files deeply nested. 

Navigating into subfolders is taking 10-20 seconds.  I'm sure it use to be much much faster.

Any ideas what I can do to improve performance?

**Comment by tatze96** [comment]
The listing performance depends on several factors, including the number of present items (files and folders), the connection mode and whether Explorer tries to create thumbnails. The used protocol may also limit the listing performance.

The listing performance can be improved by using the [*Smart Synchronization*](https://docs.cyberduck.io/mountainduck/connect/sync/) mode and marking the wanted folders for [offline synchronization](https://docs.cyberduck.io/mountainduck/connect/sync/#keep-offline). The listing process should finish faster after the offline synchronization process is completed.
